rfc/rfc-63/rfc-63.md

```diff
@@ -0,0 +1,139 @@
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+# RFC-63: Hudi bundle standards
+
+## Proposers
+
+- @xushiyan
+
+## Approvers
+
+- @vinoth
+- @prasanna
+
+## Status
+
+JIRA: https://issues.apache.org/jira/browse/HUDI-3529
+
+## Abstract
+
+Hudi bundle jars are the user-facing artifacts that deserve special attention and careful reviews when changes happen on
+the bundle names, included jars, intended usage, etc. In this RFC, a group of standards about bundle building,
+dependency governance, and release are presented to serve as references for developers and users.
+
+## Building
```

Comment:
I think we should clearly call out our goals we're striving for in our bundling:

 - Keep bundles non-overlapping (except legacy ones) 
 - Allow engine bundles to be combined w/ other ones Cloud-specific, Catalog-specific, etc
 - Make sure bundles are compatible w/ all versions of Execution engine w/in its minor branch